src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/objects/PlayerMP.kt

```diff
@@ -16,9 +16,9 @@ import net.minecraftforge.fml.relauncher.ReflectionHelper
 
 @External
 class PlayerMP(val player: EntityPlayer) : Entity(player) {
-    val displayNameField = ReflectionHelper.findField(
-            EntityPlayer::class.java,
-            "displayname"
+    private val displayNameField = ReflectionHelper.findField(
+        EntityPlayer::class.java,
+        "displayname"
```

Comment:
While you're addressing the other feedback, you might as well move this into a companion object. I've been meaning to do this for a while (it's been so long that I actually thought I did...)